lib/Doctrine/ruleset.xml

```diff
@@ -30,4 +30,149 @@
+            <property
+                name="usefulAnnotations"
+                type="array"
+                value="
+                    @author,
+                    @dataProvider,
+                    @deprecated,
+                    @expectedException,
+                    @expectedExceptionMessage,
+                    @expectedExceptionCode,
+                    @internal,
+                    @link,
+                    @see,
+                    @since,
+                    @throws
+                "
+            />
+        </properties>
+    </rule>
+
+    <!-- Require using Throwable instead of Exception -->
+    <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/Exceptions/ReferenceThrowableOnlySniff.php"/>
+
+    <!-- Require presence of declare(strict_types=1) -->
+    <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/TypeHints/DeclareStrictTypesSniff.php">
+        <properties>
+            <property
+                name="newlinesCountBetweenOpenTagAndDeclare"
+                value="2"
+            />
+            <property
+                name="spacesCountAroundEqualsSign"
+                value="0"
+            />
+        </properties>
+    </rule>
+
+    <!-- Forbid assignments in conditions -->
+    <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/ControlStructures/AssignmentInConditionSniff.php"/>
+
+    <!-- Forbid weak comparisons -->
+    <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/ControlStructures/DisallowEqualOperatorsSniff.php"/>
+
+    <!-- Require language constructs without parentheses -->
+    <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/ControlStructures/LanguageConstructWithParenthesesSniff.php"/>
+
+    <!-- Forbid dead code -->
+    <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/Classes/UnusedPrivateElementsSniff.php"/>
+
+    <!-- Forbid unused use statements -->
+    <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/Namespaces/UnusedUsesSniff.php">
+        <properties>
+            <property name="searchAnnotations" type="boolean" value="true"/>
+        </properties>
+    </rule>
+
+    <!-- Forbid useless uses of the same namespace -->
+    <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/Namespaces/UseFromSameNamespaceSniff.php"/>
+
+    <!-- Forbid useless unreachable catch blocks -->
+    <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/Exceptions/DeadCatchSniff.php"/>
+
+    <!-- Require comma after last element in multi-line array -->
+    <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/Arrays/TrailingArrayCommaSniff.php"/>
+
+    <!-- Forbid fancy yoda conditions -->
+    <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/ControlStructures/YodaComparisonSniff.php"/>
```

Comment:
Damn right!

Reply:
This should be `SlevomatCodingStandard.ControlStructures.YodaComparison`